data/sponsors.json

```diff
@@ -0,0 +1,29 @@
+{
+    "Platinum":[
+        {
+            "alt-text":"sponsor1-border",
```

Comment:
Just name the alt-text as the sponsor name everywhere, not sponsor-border.

Reply:
Cool will make it more natural